spec/features/likes_feature_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+
+feature 'Like' do
+
+  def log_in(user)
+    visit '/'
+    click_link 'Log in'
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_button 'Log in'
+  end
+
+  before do
+    user = create :user
+    log_in(user)
+    click_link 'Add a picture'
+    attach_file('picture[image]', File.join(Rails.root,'spec',"files", 'images', 'duck.jpg'))
```

Comment:
Metrics/LineLength: Line is too long. [93/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)